ddtrace/llmobs/_llmobs.py

```diff
@@ -273,9 +292,40 @@ def annotation_context(
                         This argument is only applicable to LLM spans.
         :param name: Set to override the span name for any spans annotated within the returned context.
         """
-        return AnnotationContext(
-            cls._instance.tracer, lambda span: cls.annotate(span, tags=tags, prompt=prompt, _name=name)
-        )
+        # id to track an annotation for registering / de-registering
+        annotation_id = uuid4().hex
+
+        def get_annotations_context_id():
+            current_ctx = cls._instance.tracer.current_trace_context()
+            # default the context id to the annotation id
+            ctx_id = annotation_id
+            if current_ctx is None:
+                current_ctx = Context(is_remote=False)
```

Comment:
is there any problem to creating a "dummy" context like this just to store a baggage item?